### PR TITLE
fix: separate out android example into crate

### PR DESCRIPTION
**Pull request by littledivy**
should fix dependencies pulled in https://github.com/denoland/deno/pull/13415

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -101,7 +95,3 @@ name = "shell"
 
 [[example]]
 name = "process"
```
Comment (lucacasonato) [inline Cargo.toml:97]:
```suggestion
name = "process"

[workspace]
members = [
  "examples/android"
]
```

File: examples/android/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+target/
```
Comment (lucacasonato) [inline examples/android/.gitignore:1]: Not needed with the above change.

File: Cargo.toml (hunk 1 above)
Comment (littledivy) [reply Cargo.toml:97]: That includes it's dependency in Cargo.lock :/

Comment (lucacasonato) [reply Cargo.toml:97]: That's fine. It won't be in importers' Cargo.lock. It being in rusty_v8's lock file is desirable.

Comment (littledivy) [reply Cargo.toml:97]: Alrighty, done.

**Review by lucacasonato — APPROVED** [review]
LGTM

**Pull request merged**